From 71ed10788ba421ad846c498f8e45d1b6d4131480 Mon Sep 17 00:00:00 2001
From: Sergio Martins <smartins@kde.org>
Date: Sat, 4 Feb 2017 17:07:56 +0000
Subject: [PATCH] Bake clazy into clang so that it works with MSVC

clang plugins are not very well supported on Windows. Even though
there were some patches recently I can't get clazy to work, and not
even clang's example plugins work out of the box.
---
 tools/driver/CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tools/driver/CMakeLists.txt b/tools/driver/CMakeLists.txt
index e03b3fa..65085f6 100644
--- a/tools/driver/CMakeLists.txt
+++ b/tools/driver/CMakeLists.txt
@@ -24,10 +24,13 @@ if(CLANG_PLUGIN_SUPPORT)
   set(LLVM_NO_DEAD_STRIP 1)
 endif()
 
+include(${CMAKE_CURRENT_SOURCE_DIR}/clazy/ClazyMSVCSupport.cmake)
+
 add_clang_tool(clang
   driver.cpp
   cc1_main.cpp
   cc1as_main.cpp
+  ${CLAZY_SRCS} ${CLAZY_LIB_SRC}
   )
 
 target_link_libraries(clang
-- 
2.8.0.windows.1

